xc7/devices/CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ add_conda_pip(
 list(APPEND DEVICE_RR_PATCH_DEPS intervaltree textx)
 
 define_device(
-  DEVICE xc7a50t
-  ARCH artix7
-  DEVICE_TYPE xc7a50t-roi-virt
+  DEVICE xc7z010
+  ARCH zynq7
+  DEVICE_TYPE xc7z010-roi-virt
   PACKAGES test
```

Comment:
Rather than renaming these, move this to a function that takes the device and arch name